script.js

```diff
@@ -1,76 +1,121 @@
 //fixed variables
 CHOOSENUM = "Choose Number";
 ACTUALGAME = "Actual Game";
+NORMAL = "normal";
+LOWESTWINS = "lowest wins";
 
+//variables
 var player1num = 0;
 var player2num = 0;
+var player1sum = 0;
+var player2sum = 0;
 var player = 1;
 var section = CHOOSENUM;
+var gameMode = "";
 var roll1 = 0;
 var roll2 = 0;
 
-//variables
+//message
+var chooseOrderMessage = function (player, roll1, roll2) {
+  return `Player ${player}, please choose the order which order you want. Either <br><br> 1: ${roll1} before ${roll2} <br> 2: ${roll2} before ${roll1}`;
+};
```

Comment:
[Comment] Nice helper function so that you don't have to retype for every since player. =)